runtime/doc/api.txt

```diff
@@ -1797,81 +1798,93 @@ nvim_buf_get_extmark_by_id({buffer}, {ns_id}, {id})
+                defined by buffer positions (inclusive, 0-indexed
+                |api-indexing|).
+
+                Region can be given as (row,col) tuples, or valid extmark ids
+                (whose positions define the bounds). 0 and -1 are understood
+                as (0,0) and (-1,-1) respectively, thus the following are
+                equivalent:
+>
+                  nvim_buf_get_extmarks(0, my_ns, 0, -1, {})
+                  nvim_buf_get_extmarks(0, my_ns, [0,0], [-1,-1], {})
+<
 
-                all_marks = nvim_buf_get_extmarks(0, my_ns, 0, -1, {})
+                If `end` is less than `start` , traversal works backwards.
+                (Useful with `limit` , to get the first marks prior to a given
+                position.)
 
-                If end is a lower position than start, then the range will be
-                traversed backwards. This is mostly useful with limited
-                amount, to be able to get the first marks prior to a given
-                position.
+                Example:
+>
+                  local a   = vim.api
+                  local pos = a.nvim_win_get_cursor(0)
+                  local ns  = a.nvim_create_namespace('my-plugin')
+                  -- Create new extmark at line 1, column 1.
+                  local m1  = a.nvim_buf_set_extmark(0, ns, 0, 0, 0, {})
+                  -- Create new extmark at line 3, column 1.
+                  local m2  = a.nvim_buf_set_extmark(0, ns, 0, 2, 0, {})
+                  -- Get extmarks only from line 3.
+                  local ms  = a.nvim_buf_get_extmarks(0, ns, {2,0}, {2,0}, {})
+                  -- Get all marks in this buffer + namespace.
+                  local all = a.nvim_buf_get_extmarks(0, ns, 0, -1, {})
+                  print(vim.inspect(ms))
+<
 
                 Parameters: ~
-                    {buffer}  The buffer handle
-                    {ns_id}   An id returned previously from
-                              nvim_create_namespace
-                    {start}   One of: extmark id, (row, col) or 0, -1 for
-                              buffer ends
-                    {end}     One of: extmark id, (row, col) or 0, -1 for
-                              buffer ends
-                    {opts}    additional options. Supports the keys:
-                              • amount: Maximum number of marks to return
+                    {buffer}  Buffer handle, or 0 for current buffer
+                    {ns_id}   Namespace id from |nvim_create_namespace()|
+                    {start}   Start of range, given as (row, col) or valid
+                              extmark id (whose position defines the bound)
+                    {end}     End of range, given as (row, col) or valid
+                              extmark id (whose position defines the bound)
+                    {opts}    Optional parameters. Keys:
+                              • limit: Maximum number of marks to return
```

Comment:
@norcalli JFI in case you are using the `amount` param: renamed it to `limit`.

Reply:
@justinmk thanks :)